plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/AnalyzeHandle.java

```diff
@@ -28,17 +28,20 @@
     private final boolean initialAnalyze;
     private final Optional<Instant> filesModifiedAfter;
     private final Optional<Set<String>> columns;
+    private final Optional<Boolean> recalculateAllStatistics;
 
     @JsonCreator
     public AnalyzeHandle(
             @JsonProperty("initialAnalyze") boolean initialAnalyze,
             @JsonProperty("startTime") Optional<Instant> filesModifiedAfter,
-            @JsonProperty("columns") Optional<Set<String>> columns)
+            @JsonProperty("columns") Optional<Set<String>> columns,
+            @JsonProperty("recalculateAllStatistics") Optional<Boolean> recalculateAllStatistics)
```

Comment:
Can we use initialAnalyze instead of adding new field?

Reply:
Could we have an enum mode with `INITIAL`, `INCREMENTAL`, and `FULL_REFRESH` options?